### PR TITLE
Add AnchorJS for deep linking project titles

### PR DESCRIPTION
**Pull request by IshaDakota**
see #18 ... with directory structure change, easier to just start over.

@amitaibu I guess installing bower components probably needs to be added to the deployment?

### CODE REVIEW

File: src/assets/javascript/script.js [hunk 1]
```diff
@@ -11,3 +11,8 @@ $(function() {
   // Add Semantic UI related classes to tables.
   $('table').addClass('ui table');
 });
+
+$(function() {
+  // Add deep anchor links to post pages.
+  anchors.add('.projects .project-title');
```
Comment (amitaibu) [inline src/assets/javascript/script.js]: move below the above `$('table')...` command, which is already wrapped by a `$(function() {`

**Comment by amitaibu** [comment]
> I guess installing bower components probably needs to be added to the deployment?

Can you please confirm this is not already the case `gulp serve:prod`. If not, please open a follow up.

**Comment by IshaDakota** [comment]
@amitaibu for review again.

**Comment by amitaibu** [comment]
Looks good, thanks. I've also added anchor to blog posts.

**Pull request merged**